HealthcareSystem/PatientWebApp/wwwroot/js/activate_account.js

```diff
@@ -10,7 +10,7 @@
             success: function () {
 
                 setTimeout(function () {
-                    window.location.href = 'patients_home_page.html';
+                    window.location.href = 'login.html';
                 }, 500);
```

Comment:
Maybe a good improvement for the future could be to make the activation endpoint do login automatically as well.

Reply:
I think that our product owner requested redirection to the login page after the successful activation of the account.